MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include twisted_iocpsupport/*.pxi
 include twisted_iocpsupport/*.pyx
 exclude twisted_iocpsupport/*.c
-include twisted_iocpsupport/winsock_pointers.c
+include twisted_iocpsupport/*.h
```

Comment:
we don't actually want to include any Cython generated .h files because these are built with PEP517 in the sdist->wheel phase

https://github.com/twisted/twisted-iocpsupport/pull/3